src/Microsoft.HttpRepl.Tests/Commands/GetCommandTests.cs

```diff
@@ -97,5 +97,85 @@ public async Task ExecuteAsync_WithOnlyBaseAddress_VerifyOutput()
             Assert.Contains("HTTP/1.1 200 OK", result);
             Assert.Contains(expectedResponse, result);
         }
+
+        [Fact]
+        public async Task ExecuteAsync_WithJsonContentTypeInHeader_FormatsResponseContent()
+        {
+            string json = @"{
+""swagger"": ""2.0"",
+""info"": {
+""version"": ""v1""
+},
+""paths"": {
+""/api/Employees"": {
+}
+}
+}";
+            string path  = "this/is/a/test/route/for/formatting";
+
+            _urlsWithResponse.Add(_baseAddress + path, json);
+
+            ArrangeInputs(commandText: "GET",
+                baseAddress: _baseAddress,
+                path: path,
+                urlsWithResponse: _urlsWithResponse,
+                out MockedShellState shellState,
+                out HttpState httpState,
+                out ICoreParseResult parseResult,
+                out MockedFileSystem fileSystem,
+                out IPreferences preferences,
+                contentType: "application/json");
+
+            GetCommand getCommand = new GetCommand(fileSystem, preferences);
+            await getCommand.ExecuteAsync(shellState, httpState, parseResult, CancellationToken.None);
+
+            string expectedResponse = @"{
+  ""swagger"": ""2.0"",
+  ""info"": {
+    ""version"": ""v1""
+  },
+  ""paths"": {
+    ""/api/Employees"": {
+    }
+  }
+}";
+            List<string> result = shellState.Output;
+
+            Assert.Equal(3, result.Count);
+            Assert.Contains("HTTP/1.1 200 OK", result);
```

Comment:
Should we be asserting about the order of these?  E.g. use `Assert.Equal("HTTP/1.1 200 OK", result[0]);`

Reply:
Sure, updated here:
https://github.com/aspnet/HttpRepl/commit/11d66469fca28c5150c36e837ba5f754d35f38f1